assets/js/renderer.js

```diff
@@ -388,8 +388,32 @@ $(document).ready(function () {
     // Center and reset node view
     resetZoomButton.click(() => nodeView.centerNode());
 
+    // Instantiate the splitter.
+    const splitter = Split(['#leftCol', '#tabbar'], {
+        sizes: [18, 76],
+        minSize: 200,
+        gutterSize: 5,
+        onDragEnd: function() {
+            sessionStorage.setItem('splitSizes', JSON.stringify(splitter.getSizes()));
+            barChart.draw();
```

Comment:
Nice! However, we should only redraw when the user is in the chart summary view. Currently, it performs the DB calls and draw even when the chart view is not displayed.